tests/pdb_test.go

```diff
@@ -0,0 +1,134 @@
+package tests_test
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	expect "github.com/google/goexpect"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	kvpointer "kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+var _ = Describe("[sig-compute]Pod Disruption Budget (PDB)", decorators.SigCompute, func() {
+
+	It("should ensure the Pod Disruption Budget (PDB) is deleted when the VM is stopped via guest OS", func() {
+		var err error
+
+		By("Creating test VM")
+		vm := &v1.VirtualMachine{
```

Comment:
Isn't smth like this enough?
```
		vm := libvmi.NewVirtualMachine(
			libvmifact.NewCirros(
				libvmi.WithEvictionStrategy(v1.EvictionStrategyLiveMigrate),
				libvmi.WithResourceMemory("128Mi"),
			),
			libvmi.WithRunStrategy(v1.RunStrategyManual),
		)
```

Reply:
true, i actually searched for a way to build vm from vmi and couldn't grasp that this function exist.. that's a huge help !